gulpfile.babel.js

```diff
@@ -72,17 +72,15 @@ gulp.task('coveralls', () => {
     .pipe($.coveralls())
 })
 
-gulp.task('test', (cb) => seq('lint', 'coverage', 'coveralls', cb))
+gulp.task('test', (cb) => seq('lint', 'flow', 'coverage', 'coveralls', cb))
 
 gulp.task('watch', () => gulp.watch('src/**/*', ['build']))
 
-gulp.task('doc:clean', () => del('docs'))
+gulp.task('flow', async () => {
```

Comment:
Is this `async` the only reason why you added stage-3 and the runtime transform, or is there another transitive dependency somewhere? If not, I'd just leave out the `async` because the end result will be the same. Also, maybe we want to drop Gulp altogether in favor of npm scripts?